static/install/web.html

```diff
@@ -35,6 +35,16 @@
     <body>
         {% include "header.html" %}
         <main id="web-install">
+            <p id="sticky-error-text" hidden="hidden">Your device doesn't have enough free storage space.
```

Comment:
Suggestion: something like: "Your browser reports that your device doesn't have enough free storage space to install GrapheneOS (needed: 32 gigabytes).  This may be caused by insufficient storage being free on your device, or by some artificial restriction, such as an "Incognito mode" tab or an "InPrivate Window".  Please close this tab/window, then check that your device has enough free storage space, then start over, without "Incognito mode", "InPrivate Window", etc.  Also, please check that all necessary #prerequisites are satisfied."